implementations/elixir/ockam/ockam/lib/ockam/transport/uds/address.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Ockam.Transport.UDSAddress do
```

Comment:
As mentioned in other comment, we're trying to steer away from specific address types, so let's not add new ones.